src/proc.ts

```diff
@@ -149,12 +153,14 @@ export function execute(command: string, args?: string[], outputConsumer?: Outpu
             resolve({ retc: 0, stdout: '', stderr: '' });
         });
     } else {
+        // Since we won't be sending anything to this process, close its stdin.
+        spawn_opts.stdio = ['ignore', 'pipe', 'pipe'];
```

Comment:
maybe let's remove this since this `execute` is a generic function, not just for querying compiler versions.